package.json

```diff
@@ -67,13 +67,9 @@
     },
     "devDependencies": {
         "@babel/core": "^7.23.5",
-        "@babel/plugin-proposal-optional-chaining": "^7.21.0",
         "@babel/plugin-transform-modules-commonjs": "^7.23.3",
         "@babel/preset-env": "^7.23.5",
         "@babel/runtime": "^7.23.5",
-        "babel-plugin-add-module-exports": "^1.0.4",
-        "babel-plugin-module-resolver": "^5.0.0",
-        "babel-plugin-transform-import-meta": "^2.2.1",
```

Comment:
Removed these packages because it not used here?

Reply:
yes, exactly